### PR TITLE
Feat/cross platform with tests

### PR DESCRIPTION
**Pull request by verberktstan**
- [x] Move files from src/clj/easings/ to src/easings/
- [x] Add npm & shadow-cljs
- [x] Configure :tests build
- [x] Implement basic unit tests in easings/core-test ns
- [x] Make clj work again xD
- [x] Add clj tests as well
- [x] Configure shadow via lein project.clj

### CODE REVIEW

**Comment by verberktstan** [comment]
This assumes java 17 (or higher) to run the shadow-cljs command(s). Maybe annotate this assumption in the README?

**Comment by verberktstan** [comment]
Fixes issue #1

**Comment by verberktstan** [comment]
@sdedovic We're good to go, I think! We can merge this into your feat/cross-platform branch. And then we can merge that into master :-)

**Comment by verberktstan** [comment]
And, i'm already working this to a uniform interface with quil.core/lerp for my personal use;
https://github.com/verberktstan/genartgear/commit/269e6090d31a33d722ba4fda2ee98fa16c8f690b

Good times :partying_face:

File: project.clj [hunk 1]
```diff
@@ -4,11 +4,12 @@
...
-  :resource-paths ["resources"]
-
-  :profiles {:examples {:dependencies [[quil "4.3.1563"]]
-                        :source-paths ["src/clj" "examples"]}}
+  :source-paths ["src" "examples"]
+  :java-source-paths ["src"]
+  :profiles {:examples {:dependencies [[quil/quil "4.3.1563"]]
+                        :source-paths ["src" "examples"]}
+             :cljs {:source-paths ["src" "test"]
+                    :dependencies [#_[org.clojure/clojurescript "1.11.123"] ;; TODO: Make cljs dep explicit
```
Comment (sdedovic) [inline project.clj]: what needs to be done for this?

Comment (verberktstan) [reply project.clj]: Somehow, when i tried to use an explicit cljs version i got this:
```IllegalArgumentException: No matching method legacySetOutputFeatureSet found taking 1 args for class com.google.javascript.jscomp.CompilerOptions```

But i found a workaround in [shadow-cljs project.clj](https://github.com/thheller/shadow-cljs/blob/master/project.clj)

And now it works. So i'll commit this as well :-)

**Review by sdedovic — APPROVED** [review]

**Comment by verberktstan** [comment]
Great to collaborate @sdedovic, keep up the good work!

**Comment by sdedovic** [comment]
> And, i'm already working this to a uniform interface with quil.core/lerp for my personal use; [verberktstan/genartgear@269e609](https://github.com/verberktstan/genartgear/commit/269e6090d31a33d722ba4fda2ee98fa16c8f690b)
> 
> Good times 🥳

oh very cool @verberktstan, I have my own toolkit here, feel free to poke around and use anything: https://github.com/sdedovic/artlib-core

**Comment by sdedovic** [comment]
do you post your art anywhere?

**Comment by verberktstan** [comment]
I posted some on Insta a few years ago. Just revived my website, it's a WIP very much, but hey, at least i'm rolling...

http://stanverberkt.nl/

I know, lots of things to do, https, contact info, some more info and contact etc etc. To be continued!

**Pull request merged**